packages/backend-modules/contributors/migrations/20250703152810-create-contributors-up.sql

```diff
@@ -0,0 +1,39 @@
+-- Create ENUM types for gender and employee status
+CREATE TYPE gender_enum AS ENUM ('m', 'f', 'd');
+CREATE TYPE employee_status_enum AS ENUM ('past', 'present');
+
+-- Create the contributors table
+CREATE TABLE contributors (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    name TEXT NOT NULL,
+    slug TEXT NOT NULL UNIQUE,
+    short_bio TEXT,
+    image TEXT,
+    prolitteris_id TEXT UNIQUE,
+    prolitteris_name TEXT,
+    gender gender_enum,
+    user_id UUID REFERENCES users(id),
```

Comment:
@lucianak Unsure here, if the UUID should be a reference or not. If we assume that publikator shouldn't know anything about our users, this should just be a string, right? Which unfortunately means we won't be able to validate if the correct user ID is added.